visualization_plot_f1_scores.py

```diff
@@ -0,0 +1,167 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py:percent
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.11.1
+#   kernelspec:
+#     display_name: Python 3
+#     language: python
+#     name: python3
+# ---
+
+# %% [markdown]
+# # Plot f1-scores of several models #43
+# Issue link: https://github.com/dominikmn/one-million-posts/issues/43
+
+# %%
+from transformers import pipeline
+import pandas as pd
+import numpy as np
+from tqdm import tqdm
+from utils import loading, feature_engineering, scoring
+from sklearn.metrics import precision_score, recall_score, f1_score
+import seaborn as sns
+import plotly.express as px
```

Comment:
you are not using express and could remove the import. Same with seaborn, tqdm , pipeline, pio, and plt.

Reply:
Done